client/components/vertical-menu/README.md

```diff
@@ -0,0 +1,40 @@
+# Vertical Menu
+
+This component provides a vertically-oriented list of items for typical use in selecting between different views in another part of the visible screen.
```

Comment:
If this component is specific to the sidebar as @folletto suggests, then maybe we should mention that here.